src/api/environment.cc

```diff
@@ -703,6 +700,18 @@ Maybe<void> InitializeContextRuntime(Local<Context> context) {
   // to the runtime flags, propagate the value to the embedder data.
   bool is_code_generation_from_strings_allowed =
```

Comment:
The `V8Option{}` is removed in `node_options.cc` for `--disallow-code-generation-from-strings`, this will always be `true`.